### PR TITLE
[MCHECKSTYLE-414] - documentation: property expansion example does not parse

### PR DESCRIPTION
**Pull request by pearceeverydaylabs**
Following this checklist to help us incorporate your 
contribution quickly and easily:

 - [x] Make sure there is a [JIRA issue](https://issues.apache.org/jira/browse/MCHECKSTYLE) filed 
       for the change (usually before you start working on it).  Trivial changes like typos do not 
       require a JIRA issue.  Your pull request should address just this issue, without 
       pulling in other changes.

Jira: [MCHECKSTYLE-414](https://issues.apache.org/jira/browse/MCHECKSTYLE-414)
I did include an extra example of how to use multiple properties in maven without the use of the properties file. Lmk if I should split this out into a separate PR

 - [x] Each commit in the pull request should have a meaningful subject line and body.
 - [x] Format the pull request title like `[MCHECKSTYLE-XXX] - Fixes bug in ApproximateQuantiles`,
       where you replace `MCHECKSTYLE-XXX` with the appropriate JIRA issue. Best practice
       is to use the JIRA issue title in the pull request title and in the first line of the 
       commit message.
 - [x] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
 - [ ] Run `mvn clean verify` to make sure basic checks pass. A more thorough check will 
       be performed on your pull request automatically.
 - [ ] You have run the integration tests successfully (`mvn -Prun-its clean verify`).

If your pull request is about ~20 lines of code you don't need to sign an
[Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf) if you are unsure
please ask on the developers list.

To make clear that you license your contribution under 
the [Apache License Version 2.0, January 2004](http://www.apache.org/licenses/LICENSE-2.0)
you have to acknowledge this by using the following check-box.

 - [x] I hereby declare this contribution to be licenced under the [Apache License Version 2.0, January 2004](http://www.apache.org/licenses/LICENSE-2.0)

 - [x] In any other case, please file an [Apache Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

### CODE REVIEW

File: src/site/apt/examples/custom-property-expansion.apt.vm [hunk 1]
```diff
@@ -68,7 +68,10 @@ Using Custom Checkstyle Property Expansion Definitions
         <version>${project.version}</version>
         <configuration>
           <configLocation>checkstyle.xml</configLocation>
-          <propertyExpansion>projectname=whizbang</propertyExpansion>
+          <propertyExpansion>
+            projectname=whizbang
+            anotherProperty=bazinga
+          </propertyExpansion>
```
Comment (slawekjaranowski) [inline src/site/apt/examples/custom-property-expansion.apt.vm:74]: oh, this parameter should have `Map` type instead `String` ... but it is another story 😄

Comment (pearceeverydaylabs) [reply src/site/apt/examples/custom-property-expansion.apt.vm:74]: ah! Ya, I didn't dig in to look at the code (tho I'd like to when I have the time), I just figured this out locally and figured I'd leave it as a breakcrumb for other people with the same question :)

Comment (slawekjaranowski) [reply src/site/apt/examples/custom-property-expansion.apt.vm:74]: we can deprecate `propertyExpansion` and add new 

```
Map<String, String> propertyExpansions;
```

then we can use in configuration:

```
<propertyExpansions>
 <key1>value</key1>
 <key2>value</key2>
</propertyExpansions>

```

if wish please create issue for it.

**Comment by jira-importer** [comment]
Resolve #326

**Pull request merged**